docs/annotations.md

```diff
@@ -96,3 +97,89 @@ class MyControl extends Control
     }
 }
 ```
+
+## `@phpstan-latte-template`
+Allowed context: render call, method, class
+
+Value specifies what template or templates should be resolved and checked in given context.
+
+Multiple annotations can be used on single element or at different levels to check multiple tempaltes with same context.
```

Comment:
you have always typo in word templates :) but only in discussions / readme, in code it is usually correct

Reply:
Yes I have few such typos in muscle memory and cannot get rid of them